lib/assets/chart_cell/main.css

```
@@ -167,3 +194,77 @@ input::-webkit-inner-spin-button {
   -webkit-appearance: none;
   margin: 0;
 }
+
+.accordion-control {
+  background-color: var(--gray-100);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 0.875rem;
+  padding: 0.25rem 0.5rem;
+  border-radius: 7px;
+  width: auto;
+  flex-grow: 1;
+  color: var(--gray-500);
+}
+
+.control--sm {
+  background-color: transparent;
+  position: relative;
+  top: 42px;
+  left: 75%;
+  height: 0px;
+  width: 24%;
+}
+
+.accordion-svg {
+  transition-property: all;
+  transition-duration: 200ms;
+  width: 0.75rem;
+  height: 0.75rem;
+  color: var(--gray-600);
+  margin-left: 0.75rem;
+  padding: 10px;
+  background-color: var(--gray-100);
+  border: 1px solid var(--gray-200);
+  border-radius: 5px;
+}
+
+.accordion-svg:hover {
+  background-color: var(--gray-300);
+  cursor: pointer;
+}
+
+.rotate-180 {
+  transform: rotate(180deg);
+}
+
+.rotate-0 {
+  transform: rotate(0deg);
+}
+
+.highlight {
+  color: var(--gray-600);
+}
+
+.layer-wrapper {
+  border-bottom: 1px solid var(--gray-100);
+  margin-left: 14px;
+  margin-right: 14px;
+  margin-bottom: 12px;
```

Even spacing above and below the line:

```suggestion
  margin-bottom: 12px;
  padding-bottom: 8px;
```